pybump/pybump.py

```diff
@@ -174,7 +176,10 @@ def main():
         exit(1)
 
     if args['sub_command'] == 'get':
-        print(current_version)
+        if args['appversion']:
+          print(current_appversion)
+        else:
+          print(current_version)
```

Comment:
if the app executed with the `get` flag and `appversion` flag, on a non valid chart.yaml file, the `current_appversion` variable is not defined, and it will crash